packages/api/functions/cognito/post-authentication.js

```diff
@@ -0,0 +1,19 @@
+// import { getUser, createUser } from '../../controllers/user'
+
+exports.handler = async (event) => {
+  const {
+    sub: userId,
+    email,
+  } = event.request.userAttributes
+  console.info(`Successfully authenticated ${email} (${userId})`)
+  // const user = await getUser({ id: userId })
+
+  // if (!user) {
+  //   console.info(`User doesn't exist. Creating user with id ${userId}...`)
+  //   await createUser({
+  //     id: userId,
+  //   })
+  // }
```

Comment:
This is a no-op currently. Need to update it to insert user if they don't currently exist in Dynamo